tests/rpc.js

```diff
@@ -103,7 +126,7 @@ test('Send invite and already on project', async (t) => {
   replicate(r1, r2)
 })
 
-test('Send invite with encryption key', async (t) => {
+test('Send invite with encryption keys', async (t) => {
```

Comment:
perhaps this test should be renamed, since it used to assume that the providing some encryption keys was optional